orthoinference/src/main/java/org/reactome/orthoinference/EventsInferrer.java

```diff
@@ -92,8 +92,14 @@ public static void inferEvents(Properties props, String pathToConfig, String spe
 		String eligibleFilename = "eligible_" + species	+ "_75.txt";
 		String inferredFilename = "inferred_" + species + "_75.txt";
 		File eligibleFile = new File(eligibleFilename);
+		if (eligibleFile.exists()) {
+			eligibleFile.delete();
```

Comment:
Just curious: why delete the files? Are they so large that they take up lots of storage? Does leaving them in place cause issues with other steps?

Reply:
The script will append lines to the files, which can make it annoying to try and find out what inferences were attempted/completed on the current run

Reply:
If that's the problem, it could put separators between the sections it appends. Or it could use timestamps...

But since you're deleting the files, I'm guessing that it's not really necessary to keep them around for the long term.